### PR TITLE
fixed empty output from runtime executor in CPU flow

### PR DESCRIPTION
**Pull request by zonglinpeng**
Summary:
- add missing dependency dtype_util.cpp
- it was missing etrecord. And dump-output is not toggled in executor.
- Also enable runtime logging

Differential Revision: D64272605

### CODE REVIEW

**Review by Vysarat — APPROVED** [review]

File: backends/cadence/reference/operators/CMakeLists.txt [hunk 1]
```diff
@@ -52,6 +52,7 @@ set(_aten_ops__srcs
     "${EXECUTORCH_ROOT}/kernels/portable/cpu/op_expand_copy.cpp"
     "${EXECUTORCH_ROOT}/kernels/portable/cpu/op_gelu.cpp"
     "${EXECUTORCH_ROOT}/kernels/portable/cpu/op_empty.cpp"
+    "${EXECUTORCH_ROOT}/kernels/portable/cpu/util/dtype_util.cpp"
```
Comment (mcremon-meta) [inline backends/cadence/reference/operators/CMakeLists.txt:55]: maybe move it up with the other utils?

File: backends/cadence/aot/export_example.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 from executorch.backends.cadence.aot.compiler import (
     convert_pt2,
-    export_to_cadence,
+    export_to_cadence_edge_executorch,
```
Comment (mcremon-meta) [inline backends/cadence/aot/export_example.py:17]: is that API new? I don't see it in the changes

Comment (zonglinpeng) [reply backends/cadence/aot/export_example.py:17]: good catch. now ported.

File: backends/cadence/aot/compiler.py [hunk 3]
```diff
@@ -197,11 +204,12 @@ def export_to_edge(
 # Export the model and lower it to an EdgeProgramManager (in edge IR), and
 # apply passes specific to Cadence DSP execution. Return both to print the
 # differences.
-def export_to_cadence(
+def export_to_cadence_edge_executorch(
```
Comment (mcremon-meta) [inline backends/cadence/aot/compiler.py:207]: do we need "edge"?

**Review by mcremon-meta — APPROVED** [review]

**Pull request closed without merging**